environment.sh

```diff
@@ -68,7 +68,8 @@ else
   # Check if existing environment needs to be recreated
   echo "Existing environment found for $env_name."
   one_week_ago=$(date -d "7 days ago" '+%Y-%m-%d %H:%M:%S')
-  creation_time="$(head -n1 /home/$username/miniconda3/envs/$env_name/conda-meta/history)"
+  env_path=$CONDA_PREFIX
```

Comment:
nit: don't bother assigning $CONDA_PREFIX to a new variable - just use it directly in the creation_time definition